Azure/sonic-buildimage
# Update the Nokia sonic-pmon submodule

#### Why I did it
Updates the Nokia sonic-pmon submodule to the latest

#### How I did it
Updates the Nokia sonic-pmon submodule to the latest with the following commits:
```
    6eda3f5 Optimize the sonic_platform platform.Platform() init
    61f6da2 use ndk version wildcard
    545716d Fixed the sfp key to be "vendor_rev"
    5b17eb6 Fixed the show platform temperature command with not-present SFP issue.
``` 
#### How to verify it
Update up the image and execute CLI command "show interface transceiver eeprom".   the output should be shown correctly 

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
